idunn/blocks/commercial.py

```diff
@@ -0,0 +1,24 @@
+from .base import BaseBlock
+from typing import Literal, Optional
+
+
+class CommercialBlock(BaseBlock):
+    type: Literal["commercial"] = "commercial"
+    booking_url: Optional[str]
+    appointment_url: Optional[str]
+    order_url: Optional[str]
```

Comment:
An additional field `quotation_request_url` could be defined using the transactional link with type `QUOTATION_REQUEST`.